NeuroID/src/main/java/com/neuroid/tracker/service/NIDJobServiceManager.kt

```diff
@@ -44,18 +49,47 @@ object NIDJobServiceManager {
         return CoroutineScope(Dispatchers.IO).launch {
             while (userActive && isActive) {
                 delay(5000L)
```

Comment:
@kishisaka-nid I know this was not part of your PR, but curiousfor the whole team..do ya'll think we still need this delay(5000)? This probably increases the time to get a score...

@kevinsites @priya-xavier

Reply:
Wrong branch (was on my sensor branch). Actually it looks pretty good. Removed the delay and no GC issues it seems. Still might be good to keep in at least a 1 second delay though.

Reply:
Let me get a memory and cpu footprint of this change.

Reply:
Android 28 simulator (sample app, ENG-5802 retrofit branch)

5 sec delay
<img width="266" alt="image" src="https://github.com/Neuro-ID/neuroid-android-sdk/assets/139158682/8216e844-86a7-4d4c-900a-674a66ae1b8d">

500ms delay
<img width="292" alt="image" src="https://github.com/Neuro-ID/neuroid-android-sdk/assets/139158682/0c961536-3606-4c69-a384-40730c262ff3">

no delay
<img width="264" alt="image" src="https://github.com/Neuro-ID/neuroid-android-sdk/assets/139158682/35ef3a8d-7c48-45a2-bec1-c5e3eb12913f">

Reply:
I think we should not remove the 5 second delay.